reviews/views.py

```diff
@@ -1,3 +1,72 @@
-from django.shortcuts import render
+import boto3
+import json
+import uuid
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from core.utils     import login_required
+from my_settings    import AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY
+from users.models   import User
+from arts.models    import Art
+from reviews.models import Review, ReviewImage
+
+class ReviewView(View):
+    @login_required
+    def post(self, request, art_id):
+        try:
+            user = request.user
+            art = Art.objects.get(id=art_id)
+
+            review = Review.objects.create(
+                user_id = user.id,
+                art_id = art.id,
+                comment = request.POST.get("comment")
+            )
+
+            s3_client = boto3.client(
+                's3',
+                aws_access_key_id = AWS_ACCESS_KEY_ID,
+                aws_secret_access_key = AWS_SECRET_ACCESS_KEY
+            )
+
+            image_file = request.FILES['image_file']
+            image_file_name = uuid.uuid1().hex
+            s3_client.upload_fileobj(
+                image_file,
+                "upoonbucket",
+                image_file_name,
+                ExtraArgs={
+                    "ContentType": image_file.content_type
+                }
+            )
+
+            image_url = "https://upoonbucket.s3.us-east-2.amazonaws.com/" + image_file_name
+
+            ReviewImage.objects.create(
+                review_id = review.id,
+                image_url = image_url
+            )
+
+            return JsonResponse({'message': 'SUCCESS'}, status=201)
+        
+        except Exception as error:
+            return JsonResponse({'message': 'Error'}, status=400)
+   
+    def get(self, request, art_id):
+        try:
+            art = Art.objects.get(id=art_id)
+            reviews = Review.objects.filter(art=art)
```

Comment:
```suggestion
            reviews = Review.objects.filter(art_id=art_id)
```
art_id를 알고 있으므로, 알고 있는 id를 활용하면 좋습니다~

Reply:
수정했습니다, 감사합니다!